lib/tasks/projects.rake

```diff
@@ -15,14 +15,15 @@ namespace :projects do
     users = tenk.users.list(per_page: 100).data
 
     users.each do |user|
-      puts user.display_name
-      team_member = TeamMember.find_or_create_by!(
+      puts user
+      team_member = TeamMember.find_or_create_by!(tenk_id: user.id)
```

Comment:
Ideally the change to this line in the next commit would be included as part of this commit instead, as this will break the `name` validation.